about.html

```diff
@@ -0,0 +1,141 @@
+        invitation list including organizations and individuals.
+      </p>
+    </div>
+    <div class="specificsAbout">
+      <p>Please contact us per Email for any further questions about Asia Web Conference 2022! <br />
+        <a href="mailto:balitaanmavericks@gmail.com">asiawebconf2022@awc.org</a>
+      </p>
+    </div>
+  </div>
+  <!-- Logo History section -->
+  <section class="logoHistory" id="logoHistory">
+    <div>
+      <h2>Asia Web Conference 2022 Logo</h2>
+      <img class="indicator" src="./assets/indicator.svg" alt="">
+    </div>
+    <div>
+      <p class="logoHistoryMobile">The logo Asia Web Conference 2022 was decided through the logo competition from 8th
+        of
+        March to 7th of April last
+        year.
+      <p class="logoHistoryDesktop">The logo Asia Web Conference 2022 was decided through the logo competition from 8th
+        of
+        March to 7th of April last
+        year. After online public voting and Selection Committee's judging
+        process,the logo from Regine Balagtas won the price. After consultation with Ms. Regine we optimized the
+        prize-winning
+        logo for this year's summit as following:
+      </p>
+    </div>
+    <img class="boxLogo" src="./assets/AsiaWebConf-Logo.png" alt="">
+  </section>
+  <!-- Past Events section -->
+  <section class="pastEvent" id="pastEvent">
+    <div>
+      <h2>See the past Asia Web Conferences</h2>
+      <img class="indicator" src="./assets/indicator.svg" alt="">
+    </div>
+    <p>Take a look at the last two Asia Web Conferences which took place in Cebu and Davao City.</p>
+    <ul>
+      <li class="event01">
+        <h3>2020</h3>
+        <h4>Asia Web Conference in Cebu City</h4>
+      </li>
+      <li class="event02">
+        <h3>2018</h3>
+        <h4>Asia Web Conference in Davao City</h4>
+      </li>
+    </ul>
+  </section>
+  <!-- Partners section -->
+  <section class="partner partnerMobile" id="partner">
+    <div>
+      <h2>Partner</h2>
+      <img class="indicator" src="./assets/indicator.svg" alt="">
+    </div>
+    <ul>
+      <li><img src="./assets/logoAmazon.svg" alt="">Amazon
+      </li>
+      <li><img src="./assets/logoFacebook.svg" alt="">Facebook
+      </li>
+      <li><img src="./assets/logoGoogle.svg" alt="">Google
+      </li>
+      <li><img src="./assets/logoMicrosoft.svg" alt="">Microsoft
+      </li>
+      <li><img src="./assets/logoMozilla.svg" alt="">Mozilla
+      </li>
+    </ul>
+  </section>
+  <!-- Footers section -->
+  <footer class="darkFooter">
+    <div class="logo">
+      <img src="./assets/AsiaWebConf-Logo.png" alt="">
+    </div>
+    <div>
+      <p>2022 ASIA WEB CONFERENCE. <br />All rights reserved.</p>
+    </div>
+  </footer>
+  <footer class="lightFooter">
+    <img src="./assets/AsiaWebConf-Logo.png" alt="">
+    <div>
+      <p>2022 ASIA WEB CONFERENCE. <br />All rights reserved.</p>
+    </div>
+  </footer>
```

Comment:
- [ ] Please, consider that here in the footer section of the mobile view the logo is displaying not fully. Try to fix it.
<img width="433" alt="Screen Shot 2022-04-12 at 10 29 56" src="https://user-images.githubusercontent.com/49064106/162895181-fb1ea73b-59bb-4b56-a664-ce20d2869905.png">

Reply:
@elmar8287 Hi there, it is showing fully in the mobile version. Please select a mobile device and not the responsive mode.

Reply:
@mavericks-db no footer on a mobile device also. It is required to be on the `Home` page also and should be white background 👍

Reply:
@elmar8287 Yes, I added already. Thanks 👍